collectors/ioping.plugin/metadata.yaml

```diff
@@ -33,17 +33,38 @@ overview:
       description: ''
 setup:
   prerequisites:
-    list: []
+    list: |
+      It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:
+
+      ```bash
+      /usr/libexec/netdata/plugins.d/ioping.plugin install
+      ```
   configuration:
     file:
-      name: ''
-      description: ''
+      name: 'ioping.conf'
+      description: 'File with options to specify hardware to monitor and arguments for ioping command.'
     options:
       description: ''
       folding:
-        title: ''
+        title: 'Config options'
         enabled: true
-      list: []
+      list: |
+        - name: update_every
+          description: Data collection frequency.
+          default_value: 1s
+          required: false
+        - name: ioping
+          description: The complete plugin path.
+          default_value: /usr/libexec/netdata/plugins.d/ioping
+          required: false
+        - name: request_size
+          description: The request size in bytes to ioping the destination (symbolic modifiers are supported)
+          default_value: 4k
+          required: false
+        - name: ioping_options
+          description: Options passed to `ioping` commands.
+          default_value: -T 1000000
+          required: false
     examples:
```

Comment:
@thiagoftsm @ilyam8 @Ferroin I don't know who would be best to provide a configuration example, but we need one here, and I don't know what destination should be (I guess something like /dev/sda)

Reply:
We only need to modify one parameter inside `ioping.conf`, so I pushed the minimum example and I also address other comments in last commit.